core/src/main/scala/cats/Traverse.scala

```diff
@@ -186,6 +186,28 @@ trait Traverse[F[_]] extends Functor[F] with Foldable[F] with UnorderedTraverse[
   def zipWithLongIndex[A](fa: F[A]): F[(A, Long)] =
     mapWithLongIndex(fa)((a, long) => (a, long))
 
+  /**
+   * If `fa` contains the element at index `idx`, 
+   * return the copy of `fa` where the element at `idx` is replaced with `b`. 
+   * If there is no element with such an index, return `None`. 
+   *
+   * The behavior is consistent with the Scala collection library's
+   * `updated` for collections such as `List`.
+   */
+  def updated_[A, B >: A](fa: F[A], idx: Long, b: B): Option[F[B]] = {
```

Comment:
Sorry, I didn't realize we are missing a law for this! It can be like the other laws, just verifying against the reference implementation.

Reply:
Ended up adding one of the previous implementations in https://github.com/typelevel/cats/pull/4248/commits/bdf2d4120e8d0dd9dabe7e34f45326647971aa1b. Is it okay?

Reply:
Ah, the reference implementation should be the same as the default implementation :) basically, it's a way to make sure that if someone overrides it (like we do in `Vector`) it matches what the default implementation would do.

Reply:
Btw, sorry I wrote that so confusingly, I should have just said "default implementation". I was thinking about how these laws end in `Ref` which I assume stands for "reference".

Reply:
Ah, so _that's_ why they are all the same as default. I thought it was weird that they are all just the same as the trait impl. Like, what are we even testing here? The fact that some instance may override certain default impl totally slipped from my head. Now it all makes sense, thank you!